src/js/components/ContentInfo/contentInfo.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ContentInfo from './index';
+
+describe('ContentInfo', () => {
+  it('should be defined', () => {
+    expect(ContentInfo).toBeDefined();
+  });
+  it('should render correctly', () => {
+    const wrapper = shallow(
+      <ContentInfo
```

Comment:
We usually define props right after imports: `let props = {}`, and fill it in `beforeEach` hook, then, in component you just pass them: `<Comp {...props} />`, and if you need to modify props in a test - you just do that before rendering the component: `prop.changedProp = 'newValue'; const wrapper = ...`. It helps to defined props only once and have a fresh instance of those in each test.